modules/configuring/proc-provisioning-your-custom-configuration.adoc

```diff
@@ -12,10 +12,11 @@ Your changes on this configuration might get reverted on {product-short} restart
 ====
```

Comment:
IMO, this can be a bit misleading, due to the `baseUrls` settings in the app-config.. On OCP, we can skip this configuration step because the Operator can automatically determine the defaults for the baseUrls based on the route settings. But this is currently not supported on non-OCP platforms. In that case, users must configure the correct `baseUrls` settings in the app-config.